pom.xml

```diff
@@ -628,6 +655,34 @@ limitations under the License.
         <outputDirectory>docs/reports/${project.version}</outputDirectory>
       </reporting>
     </profile>
+    <profile>
+      <id>release</id>
+      <build>
+        <plugins>
+          <!-- Skip license plugin during release as maven release is now shallow cloned -->
+          <plugin>
+            <groupId>com.mycila</groupId>
+            <artifactId>license-maven-plugin</artifactId>
+            <configuration>
+              <skip>true</skip>
+            </configuration>
+          </plugin>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-gpg-plugin</artifactId>
```

Comment:
About that, could you please add `<useAgent>true</useAgent>` in this plugin config ? I always run into the same failure and have to manually trigger the upload command again because it is missing ;-)

Reply:
@mathieucarbou Its in dependency management set to 'true' on line 407.